src/components/MainMenu/index.js

```diff
@@ -33,36 +35,36 @@ export class MainMenu extends React.Component { // eslint-disable-line react/pre
               <li>
                 <NavLink to="/" onClick={this.resetIncident}>
                   <span className="linklabel">
-                    Nieuwe melding
+                    <FormattedMessage {...messages.new_melding} />
                   </span>
                 </NavLink>
               </li>
-              {this.props.isAuthenticated ?
+              {this.props.isAuthenticated &&
                 <li>
                   <NavLink to="/manage/incidents">
                     <span className="linklabel">
-                      Afhandelen
+                      <FormattedMessage {...messages.afhandelen} />
                     </span>
                   </NavLink>
-                </li> : ''
+                </li>
               }
-              {permissions.includes('signals.sia_statusmessagetemplate_write') ?
+              {permissions.includes('signals.sia_statusmessagetemplate_write') &&
                 <li>
                   <NavLink to="/manage/standaard/teksten">
                     <span className="linklabel">
-                      Beheer standaard teksten
+                      <FormattedMessage {...messages.manage_standaard_teksten} />
                     </span>
                   </NavLink>
-                </li> : ''
+                </li>
               }
-              {/* this.props.isAuthenticated ?
+              {/* this.props.isAuthenticated &&
```

Comment:
Why not remove this block? It's not used.

Reply:
Yeah, why not